src/CfgEditor/CfgData.ts

```diff
@@ -0,0 +1,111 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as ini from 'ini';
+
+const sections = [
+  'onecc', 'one-import-tf', 'one-import-tflite', 'one-import-bcq', 'one-import-onnx',
+  'one-optimize', 'one-quantize', 'one-codegen', 'one-profile'
+];
+
+export class CfgData {
+  private _oneConfig: any = undefined;
+
+  constructor() {}
+
+  getOneConfig(): any {
+    return this._oneConfig;
+  }
+
+  getStringfied(): string {
+    return ini.stringify(this._oneConfig);
+  }
+
+  updateWithParsedConfig(cfg: any): void {
+    this._oneConfig = cfg;
+  }
+
+  updateWithStringifiedText(text: string): void {
+    this._oneConfig = ini.parse(text);
+
+    // TODO Separate handling deprecated elements
+    // NOTE 'one-build' will be deprecated.
+    //      Therefore, when only 'one-build' is used, it will be replaced to 'onecc'.
+    if (this._oneConfig['onecc'] === undefined && this._oneConfig['one-build'] !== undefined) {
+      this._oneConfig['onecc'] = ini.parse(ini.stringify(this._oneConfig['one-build']));
+      delete this._oneConfig['one-build'];
+    }
+    // NOTE 'input_dtype' is deprecated.
+    //      Therefore, when only 'input_dtype' is used, it will be replaced to 'onecc'.
+    if (this._oneConfig['one-quantize']?.['input_dtype'] !== undefined) {
+      if (this._oneConfig['one-quantize']['input_model_dtype'] === undefined) {
+        this._oneConfig['one-quantize']['input_model_dtype'] =
+            this._oneConfig['one-quantize']['input_dtype'];
+      }
+      delete this._oneConfig['one-quantize']['input_dtype'];
+    }
+  }
+
+  setParam(section: string, param: string, value: string): void {
+    if (this._oneConfig[section] === undefined) {
+      this._oneConfig[section] = {};
+    }
+    if (this._oneConfig[section][param] === undefined) {
+      this._oneConfig[section][param] = '';
+    }
+    this._oneConfig[section][param] = value;
+  }
+
+  setSection(section: string, paramStringified: string): void {
+    this._oneConfig[section] = ini.parse(paramStringified);
+  }
+
+  isSame(textStringified: string): boolean {
+    const iniDocument = ini.parse(textStringified);
+    for (const [sectionName, section] of Object.entries(this._oneConfig)) {
+      for (const [paramName, param] of Object.entries(section as any)) {
+        if (iniDocument[sectionName] !== undefined &&
+            iniDocument[sectionName][paramName] === param) {
+          continue;
+        }
+        return false;
+      }
+    }
+    for (const [sectionName, section] of Object.entries(iniDocument)) {
+      for (const [paramName, param] of Object.entries(section as any)) {
+        if (this._oneConfig[sectionName] !== undefined &&
+            this._oneConfig[sectionName][paramName] === param) {
+          continue;
+        }
+        return false;
+      }
+    }
+    return true;
+  }
+
+  getSorted(): CfgData {
```

Comment:
Function name is `getSorted` but it is doing both `get` and `set`.
So I want to suggest following idea.

```ts
  sort(): void {
    ...
  }

  const cfgData = CfgData();
  cfgData.sort();
  let someVar = cfgData.getOneConfg();
```

Reply:
Oh, that's a good point! I'll fix it soon.

Reply:
done